Website/src/rendering/src/components/News/NewsGrid.tsx

```diff
@@ -11,8 +16,15 @@ type NewsGridProps = ComponentProps & {
 };
 
 const NewsGrid = (props: NewsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasNews = props.fields;
```

Comment:
`hasNews` is an object with that code. We could use `!!props.fields` for it to be a boolean and consume less memory.